notebooks/00-getting_started.ipynb

```diff
@@ -194,7 +172,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "prediction = axon_segmentation([path_testing], [\"image.png\"], path_model, config_network, resampled_resolutions=0.1, verbosity_level=3)"
+    "segment_image(\n",
+    "        path_testing_image=path_testing / 'image.png',\n",
+    "        path_model=path_model,\n",
+    "        overlap_value=[48, 48],\n",
+    "        acquired_resolution=0.13\n",
```

Comment:
The previous version with the axon_segmentation function was able to extract the pixel size from the txt file [here](https://github.com/neuropoly/axondeepseg/blob/45e7da020fd7b1e51ba84089252536a3619d98a4/AxonDeepSeg/apply_model.py#L238).
The new version of axon_segmentation does not have that feature so we need the acquired_resolution here. Would it be possible to bring back the feature in axon_segmentation so the resolution is extracted from the txt file instead?

Reply:
Resolved in 0438ad28f5f4a1d

Reply:
I tested it by removing the "acquired_resolution=0.13" in the "segment_image" call but I got an error, I tried a couple of thing but don't understand what is not working:
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Input In [7], in <module>
----> 1 segment_image(
      2         path_testing_image=path_testing / 'image.png',
      3         path_model=path_model,
      4         overlap_value=[48, 48])

File ~/code/axondeepseg/AxonDeepSeg/segment.py:112, in segment_image(path_testing_image, path_model, overlap_value, acquired_resolution, verbosity_level)
    109 img_name_original = acquisition_name.stem
    111 # Performing the segmentation
--> 112 axon_segmentation(path_acquisitions_folders=path_acquisition, acquisitions_filenames=[str(path_acquisition / acquisition_name)],
    113                   path_model_folder=path_model, overlap_value=overlap_value,
    114                   acquired_resolution=acquired_resolution)
    116 if verbosity_level >= 1:
    117     print(("Image {0} segmented.".format(path_testing_image)))

File ~/code/axondeepseg/AxonDeepSeg/apply_model.py:35, in axon_segmentation(path_acquisitions_folders, acquisitions_filenames, path_model_folder, overlap_value, acquired_resolution, verbosity_level)
     33 if (path_acquisitions_folders / 'pixel_size_in_micrometer.txt').exists():
     34     resolutions_file = open(path_acquisitions_folders / 'pixel_size_in_micrometer.txt', 'r')
---> 35     str_resolution = [float(file_.read()) for file_ in resolutions_file]
     36     acquired_resolution = float(str_resolution[0])
     37 else:

File ~/code/axondeepseg/AxonDeepSeg/apply_model.py:35, in <listcomp>(.0)
     33 if (path_acquisitions_folders / 'pixel_size_in_micrometer.txt').exists():
     34     resolutions_file = open(path_acquisitions_folders / 'pixel_size_in_micrometer.txt', 'r')
---> 35     str_resolution = [float(file_.read()) for file_ in resolutions_file]
     36     acquired_resolution = float(str_resolution[0])
     37 else:

AttributeError: 'str' object has no attribute 'read'
```